utils/utils.go

```diff
@@ -6,18 +6,35 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"runtime"
 	"sort"
 	"strings"
 
 	"github.com/go-git/go-git/v5"
 )
 
 func RepoDir() string {
+	const gitignoreDir = ".gitignore-boilerplates"
+
 	override := os.Getenv("GIBO_BOILERPLATES")
 	if len(override) > 0 {
 		return override
 	}
-	return filepath.Join(os.Getenv("HOME"), ".gitignore-boilerplates")
+
+	override = os.Getenv("XDG_DATA_HOME")
+	if len(override) > 0 {
+		return filepath.Join(override, "gibo", gitignoreDir)
```

Comment:
Sorry for the drive-by review, but I'd like to point out that directories in `$XDG_DATA_HOME`, `$XDG_CONFIG_HOME`, etc. should not be hidden. So it should be something like `path.join(XDG_DATA_HOME, 'gibo', 'gitignore-boilerplates')`